integration_tests/cancel_basic_workflow.t

```diff
@@ -24,6 +24,12 @@ $wf_proxy->cancel;
 $wf_proxy->wait(polling_interval => 1);
 is($wf_proxy->status, 'canceled', 'Got expected (canceled) status');
 is_deeply($wf_proxy->outputs, undef, 'Got expected (empty) outputs');
+
+# this keeps us from deleting the workflow before petri can send
+# 'set_dag_status_running' callbacks
```

Comment:
It's just noise, all the tests have already finished by this time.  Additionally, it returns 500 which is retried by the petri service making it noise that persists.